crates/rate-limiter/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "rate-limiter"
```

Comment:
nanonit: I have a slight preference for `rate-limit` over `rate-limiter` as I feel like our newer crate names describe more what they do rather then the main component they are.